RLTest/env.py

```diff
@@ -168,18 +172,7 @@ def __init__(self, testName=None, testDescription=None, module=None,
             print(Colors.Gray('\tdescription: ' + testDescription))
 
         self.module = module if module else Defaults.module
-        self.moduleArgs = None
-        if moduleArgs:
-            self.moduleArgs = moduleArgs
-        if Defaults.module_args:
-            if not self.moduleArgs:
-                self.moduleArgs = Defaults.module_args
-            else:
-                defaultArgs = Defaults.module_args.split(' ')
-                for i in range(0, len(defaultArgs) - 1, 2):
-                    # join module args
-                    if defaultArgs[i] not in self.moduleArgs:
-                        self.moduleArgs += ' %s %s' % (defaultArgs[i], defaultArgs[i + 1])
+        self.moduleArgs = moduleArgs if moduleArgs else Defaults.module_args
```

Comment:
@filipecosta90 did we remove this on purpose?

Reply:
Yes, we're handling the logic within the redis_std. Tested in https://github.com/RedisLabsModules/RLTest/pull/99/files#diff-4ecaac2392eec4d49d6792a41f1beaa96f57501d451c4f96a6604ed0594354fbR29